### PR TITLE
docs: instructions for deploying a wormchain validator added.

### PR DESCRIPTION
**Pull request by qedgardo**

### CODE REVIEW

**Review by ncavedale-xlabs — APPROVED** [review]
LGTM

File: docs/operations.md [hunk 1]
```diff
@@ -107,7 +107,37 @@ Refer to the [Terra Classic documentation](https://classic-docs.terra.money/docs
...
+```bash
+make build/wormchaind
+```
+You'll find `wormchaind` binary in `wormchain/build/wormchaind"`.
+
+#### Initialize environment
+Initialize the Wormchain environment with the necessary configurations.
+
+```bash
+/path/to/wormchain/bin/wormchaind init <your-node-name> --chain-id wormchain --home /path/to/wormchain/directory
```
Comment (andreclaro) [inline docs/operations.md:134]: Is this `your-node-name` important? should this be defined by WF or defined by the guardian and shared with WF?

Comment (qedgardo) [reply docs/operations.md:134]: IIUC it's the moniker. In our case is "xLabs", so maybe would be a good idea to update the docs with <guardian-name>. WDYT?

Comment (andreclaro) [reply docs/operations.md:134]: yes!

```
moniker - A custom human readable name for this node
```

what's the impact and rules for setting this name?

**Pull request merged**